src/app/grid/grid-sample-selection-template-excel/grid-sample-selection-template-excel.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="grid__wrapper">
+    <igx-grid #myGrid [data]="data" [primaryKey]="'ID'" [rowSelection]="'multiple'" [cellSelection]="'none'"
+        [paging]="true" [height]="'530px'" width="100%">
+        <igx-column [field]="'ContactName'" [width]="'20%'"></igx-column>
```

Comment:
Applies for everything:
```suggestion
        <igx-column field="ContactName" width="20%"></igx-column>
```